ui/app/styles/components/tabs.scss

```diff
@@ -15,25 +19,25 @@
       border-color: $blue;
       color: $blue;
     }
-    &:first-child a,
-    &:first-child .tab {
-      margin-left: $size-5;
-    }
   }
 
   a,
   .tab {
-    color: $grey-dark;
+    color: $grey;
     font-weight: $font-weight-semibold;
     text-decoration: none;
     padding: $size-6 $size-8 $size-8;
     border-bottom: 2px solid transparent;
-    transition: border-color $speed;
+    transition: background-color $speed, border-color $speed;
 
     &:hover,
     &:active {
       border-color: $grey-light;
     }
+
+    &:hover {
+      background-color: $ui-gray-050;
```

Comment:
This hover state has been a part of the design system for a while but hadn't been updated yet in Vault